### PR TITLE
Fixes and Weblate mentioned in README

### PR DESCRIPTION
**Pull request by comradekingu**
Relevant to #14

### CODE REVIEW

**Comment by braniii** [comment]
Thx for your suggestions to improve the Readme. I am busy this week, but will try to review it asap. @gwosd or do you have time?

And as a remark for myself: In general I would translate these changes to the fastlane metadata. And remove there the unnecessary line breaks.

File: README.md [hunk 1]
```diff
@@ -17,19 +18,20 @@ Join our community today to trace yourself 🐺🤸‍♀️🏋‍♀️🧘‍
...
-- Truly **FREE**: No annoying advertising no permission
-- Modern appearance using **MATERIAL DESIGN 3**
-- **CUSTOMIZABLE**: Many free custom themes, **DARK MODE**, and several **UNITS**. Tracking your **PROGRESS**
-- Track your **PROGRESS** with state-of-the-art data modeling 🧑‍💻
-- **PRIVACY**: Not just empty promises, it is fully <a href="https://github.com/QuantumPhysique/trale">open-source</a>
+- Truly **gratis**: No annoying advertising no permission
+- Modern appearance using **Material Design 3**
+- **Customizable**: Many custom themes, **dark mode**, and several **units**. Tracking your **progress**
+- Track your **progress** with state-of-the-art data modeling 🧑‍💻
+- **Privacy**: Not just empty promises, it is fully <a href="https://github.com/QuantumPhysique/trale">copylefted libre software</a>
```
Comment (gwosd) [inline README.md:29]: I would prefer _open-source_, as the phrase _copylefted libre software_ is not widely known, and even F-Droid uses the term FOSS.

**Review by gwosd — COMMENTED** [review]
Thank a lot for improving our readme :)

Could you take my minor changes into account?

**Comment by gwosd** [comment]
@comradekingu: Thank you again for improving our readme!

**Pull request merged**